client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -2,38 +2,51 @@
  * External dependencies
  */
 import { __ as NO__ } from '@wordpress/i18n';
-import { TextControl, SelectControl } from '@wordpress/components';
+import { SelectControl } from '@wordpress/components';
 import React from 'react';
 
 /**
  * Internal dependencies
  */
 import { SiteType } from '../store/types';
 import { useOnboardingDispatch, useOnboardingState } from '../store';
+import './style.scss';
 
 export default function OnboardingEdit() {
 	const { siteTitle, siteType } = useOnboardingState();
 	const { setSiteType, setSiteTitle } = useOnboardingDispatch();
 
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
-		<>
-			<p>{ NO__( "Let's set up your website -- it takes only a moment" ) }</p>
-			{ NO__( 'I want to create a website ' ) }
-			<SelectControl< SiteType >
-				onChange={ setSiteType }
-				options={ [
-					{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
-					{ label: NO__( 'for a store.' ), value: SiteType.STORE },
-					{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
-				] }
-				value={ siteType }
-			/>
+		<div className="onboarding__questions">
+			<h2 className="onboarding__questions-heading">
+				{ NO__( "Let's set up your website – it takes only a moment." ) }
+			</h2>
+
+			<label className="onboarding__question">
+				<span>{ NO__( 'I want to create a website ' ) }</span>
+				<SelectControl< SiteType >
```

Comment:
This will likely need to be something else than pure `<select>` or `<SelectControl>`, so I'm not spending time yet to style it differently.

Reply:
For our purposes, it doesn't help to have `input` or `select`.

We might use [Reakit radio](https://reakit.io/docs/radio/) with divs and style as required, but we can investigate that later.

Reply:
`radio` is probably the best fit semantically, now that I think about it :+1: